btcec/signature.go

```diff
@@ -396,3 +401,134 @@ func RecoverCompact(curve *KoblitzCurve, signature,
 
 	return key, ((signature[0] - 27) & 4) == 4, nil
 }
+
+var one = big.NewInt(1)
```

Comment:
Please declare this in the existing variable declaration block where order and halforder are declared.